.npmignore

```diff
@@ -1,2 +1,13 @@
 test/
-tmp/
\ No newline at end of file
+tmp/
+scripts/
```

Comment:
I guess the only reason is so that they can run tests to verify their build works. It might be helpful for debugging someone's problems